### ISSUE TITLE
Motion not reported

### ISSUE THREAD

**Issue by mihaimdinca** [pergolafabio/Hikvision-Addons#156]
### What happened?

I own a Hikvision DS-KV6113-WPE1(C). I currently have the stable version 3.0.10 of the Hikvision Doorbell Home Assistant addon. I am generally happy with the addon, but the motion trigger does not work (it is always off). The Hikvision app reports motion from this device. Until recently I was using the beta version and that seemed to work fine. The log also indicates the detection of motion.

### Add-on

Hikvision Doorbell

### What version of the add-on are you running?

3.0.10

### Installation type

Home Assistant add-on

### Relevant log output

```Text
2023-10-13 22:13:32.061 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-13 22:13:32.573 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-13 22:13:33.093 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-13 22:13:33.588 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-13 22:13:34.349 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-13 22:13:34.948 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-13 22:29:54.019 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-13 23:01:56.122 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 01:15:19.531 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:55.601 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:56.112 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:56.623 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:57.132 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:57.650 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:58.161 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:58.704 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:59.215 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:46:59.721 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:00.231 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:00.740 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:01.252 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:01.761 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:02.271 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:02.781 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:03.292 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:03.938 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:04.441 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:04.952 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 02:47:05.615 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:19:19.108 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:23:33.585 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:23:34.095 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:23:35.629 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:41:31.156 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:41:31.669 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:41:32.308 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:41:32.956 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:41:33.467 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:41:33.976 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:41:34.486 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 06:45:30.616 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:30.718 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:31.231 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:31.736 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:32.250 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:32.757 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:33.266 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:33.777 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:34.288 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:35.037 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:35.600 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:36.107 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:00:36.617 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:20:17.335 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:20:53.815 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:20:54.329 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:06.103 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:06.625 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:07.137 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:07.647 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:08.157 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:08.662 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:09.185 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:26:09.683 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:28:24.869 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:28:50.864 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:28:51.372 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:28:51.881 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:29:15.850 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:29:16.361 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:29:16.874 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:32:09.018 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:32:09.519 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:40.317 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:40.826 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:41.336 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:41.847 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:42.355 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:42.867 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:43.375 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:36:44.138 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:42:29.498 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:42:30.003 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:42:30.514 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:42:31.023 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 07:42:31.584 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:05:12.141 | INFO     | mqtt_input:_reboot_callback:290 - Received reboot command for doorbell: amd_doorbell
2023-10-14 08:06:10.345 | INFO     | mqtt:video_intercom_alarm:359 - Alarm <sdk.hcnetsdk.NET_DVR_VIDEO_INTERCOM_ALARM_INFO_UNION object at 0x7faccc9140> detected on door 0
2023-10-14 08:06:10.377 | INFO     | event:video_intercom_alarm:120 - Video intercom alarm from amd_doorbell
2023-10-14 08:06:11.870 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:12.652 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:14.150 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:14.906 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:16.443 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:17.229 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:18.759 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:19.548 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:21.081 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:21.849 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
2023-10-14 08:06:23.390 | INFO     | event:motion_detection:98 - Motion detected from amd_doorbell
```

### Relevant configuration

```YAML
doorbells:
  - name: amd_doorbell
    ip: 192.168.1.77
    username: admin
    password: ******
system:
  log_level: INFO
  sdk_log_level: NONE
mqtt:
  host: 192.168.1.***
  port: 1883
  ssl: false
  username: ********
  password: ********
```

**Comment by pergolafabio** [comment]
That's strange, should be no difference in the stable version , maybe the entities are messed up...

Can you stop the addon, delete the whole mqtt device, so all entities are removed, then start addon again?

**Comment by mihaimdinca** [comment]
I managed to get it working. Apparently the problem was with the automation I had running on Motion Detected trigger. After swapping beta for stable addon, I have deleted the old device and waited for a new one to be crated. Unfortunately, for some reason, after selecting the new one in the automation visual editor, the discovery ID remained the old one. After changing this, it started working again.
